juno/assetlist.json

```diff
@@ -2099,6 +2099,31 @@
           }
         }
       ]
+    },
+            {
+      "denom_units": [
+        {
+          "denom": "factory/juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq/uwind",
+          "exponent": 0
+        },
+        {
+          "denom": "wind",
+          "exponent": 6
+        }
+      ],
+      "address": "juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq",
+      "base": "factory/juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq/uwind",
+      "name": "Wind Token",
+      "display": "wind",
+      "symbol": "WIND",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
```

Comment:
The uploaded image has a capital 'W'. Could you please either correct the url or the name of the image (preferred) to make them match?